mxenabled/connect-widget
# Fixed connected screen copy

#### Issue
https://mxcom.atlassian.net/browse/CT-1131

#### Changes
- updated line breaks on connected screen
- updated language files

#### Testing Instructions
- link with MX connect
- Make sure the connected screen matches new copy
- Run cypress and vitest
